src/components/ItemTile/ItemTile.js

```diff
@@ -0,0 +1,58 @@
+import { graphql, useStaticQuery } from 'gatsby';
+import { GatsbyImage, getImage } from 'gatsby-plugin-image';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { container, content, image, text, title } from './ItemTile.module.scss';
+
+export const ItemTile = ({
+  projectTitle,
+  projectText,
+  projectImage,
```

Comment:
brakuje mi informacji w jaki sposób przekazać image do komponentu, widać useStaticQuery i nody, regexy itd, zastanawiam się czym powinien być projectImage.
Dodaj proszę komentarz jakiego rodzaju typem powinien być projectImage, oraz, jeśli to potrzebne, przykład jak skonstruować projectImage. (np. jeśli to path w postaci stringa, to w jakiej strukturze, gdzie go umieścić, jeśli obiekt, to jak zainicjalizować.